app/build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'com.android.application'
+
+repositories {
+    mavenCentral()
+    maven { url 'http://download.crashlytics.com/maven' }
```

Comment:
I'd prefer we not use Maven repositories that aren't served over HTTPS.

It looks like Crashlytics doesn't provide a Maven repository over HTTPS, but [Fabric](https://fabric.io) (its successor) does. I'll need to have an internal discussion about switching to Fabric. In the interim, could you remove this and rip out our usage of Crashlytics?